com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueContainer.cs

```diff
@@ -102,21 +96,17 @@ public void AddToInternalMLAPISendQueue(RpcFrameQueueItem queueItem)
         /// <param name="queueType"></param>
         public void ProcessAndFlushRpcQueue(RpcQueueProcessingTypes queueType, NetworkUpdateStage currentUpdateStage)
         {
-            if (m_RpcQueueProcessor == null)
-            {
-                return;
-            }
-
+            bool isListening = !ReferenceEquals(m_NetworkManager, null) && m_NetworkManager.IsListening;
```

Comment:
Again, I *think* we should just assert if we are not properly initialized with a NM

Reply:
It will assert... with an exception...if that doesn't exist... very bad things happened and it should "effectively break"...